website/blog/2020-05-25-7.10.0.md

```diff
@@ -68,23 +68,23 @@ class Person {
 }
 ```
 
-This stage 1 proposal allows to statically check if a given object has a specific private field.
+This stage 1 proposal allows you to statically check if a given object has a specific private field.
 
-Private fields have a built-in "brand check": if you try to access them in an object where they aren't defined, it will throw an exception. We can often leverage this characteristic to use a `try`/`catch` statement and detect their absence, but this proposal gives us a more compact and robust syntax to do so.
+Private fields have a built-in "brand check": if you try to access them in an object where they aren't defined, it will throw an exception. You can detect undefined private fields by leveraging this behavior with a `try`/`catch` statement, but this proposal gives us a more compact and robust syntax to do so.
```

Comment:
I don't like "undefied private fields" here, because given the presence of the `undefined` value in JS it can be easily confused with `foo.#priv === undefined`.

Reply:
Sure, initial intent was to balance "where they aren't defined"...

> You can determine if an object has a particular private field by leveraging...

Reply:
```suggestion
Private fields have a built-in "brand check": if you try to access them in an object where they aren't defined, it will throw an exception. You can determine if an object has a particular private field by leveraging this behavior with a `try`/`catch` statement, but this proposal gives us a more compact and robust syntax to do so.
```